CoreFoundation/Base.subproj/CoreFoundation_Prefix.h

```diff
@@ -236,6 +236,10 @@ CF_INLINE uint64_t mach_absolute_time() {
 #define CF_PRIVATE extern __attribute__((visibility("hidden")))
 #define __strong
 #define __weak
+
+#if defined(__OpenBSD__)
+#define strtoll_l(a,b,c,locale) strtoll(a,b,c)
```

Comment:
Could you move this with the rest of the macros ~L186 please?

Reply:
That'd conflict with the definition of strtoll_l for Windows (as _strtoi64), though.

Reply:
Okay, I think that localizing the definitions would be nicer in the future.